packages/calypso-e2e/src/lib/components/editor-block-toolbar-component.ts

```diff
@@ -0,0 +1,63 @@
+import { Locator, Page } from 'playwright';
+
+const parentSelector = '[aria-label="Block tools"]';
+
+const selectors = {
+	button: ( identifier: BlockToolbarButtonIdentifier ) => {
+		if ( ! ( identifier.ariaLabel || identifier.text ) ) {
+			throw new Error( 'You must provide at least one way to identify the menu button.' );
+		}
+
+		let selector = `${ parentSelector } button`;
+		if ( identifier.ariaLabel ) {
+			selector = `${ selector }[aria-label="${ identifier.ariaLabel }"]`;
+		}
+
+		if ( identifier.text ) {
+			selector = `${ selector }:has-text("${ identifier.text }")`;
+		}
+		return selector;
+	},
+};
+
+export interface BlockToolbarButtonIdentifier {
+	text?: string;
+	ariaLabel?: string;
+}
+
+/**
+ * Represents the popover toolbar menu that appears for a focused block.
```

Comment:
I'm thinking if it would make sense to have our test account use the `Top Toolbar` option, which would make the floating toolbar a part of the editor toolbar.

<img width="272" alt="image" src="https://user-images.githubusercontent.com/6549265/164615539-700d90a7-ebba-435a-9809-6a2611e4d467.png">

Reply:
To clarify, this is just me thinking out loud. This option appears to be unchecked by default and I'm not actually sure how 'sticky' this option is. I suspect it is not sticky at all, and is stored in the `localStorage`.

Reply:
I don't feel strongly either way! My best suggestion would be to see if we can figure out from Tracks data which is more popular, and then maybe use that approach?

The great news: even if "stick" -- the markup is actually the same (just in a different spot), and this component class would still work!

To that effect, I will remove the word "popover"